redi-connect-front/src/pages/front/signup/SignUp.tsx

```diff
@@ -0,0 +1,322 @@
+import React, { useState } from "react";
```

Comment:
Great job on this factory. Can we rename the file? Something along the lines of `sign-up-form-factory`. The current name is misleading as there is no actual Component with the name `SignUp`.

Reply:
.. just like you actually did for the other factory function, actually.

Reply:
ah, bummer knew I forgot something... thanks for the catch